src/Postmark/Transport.php

```diff
@@ -297,6 +308,22 @@ private function processHeaders(&$payload, $message) {
 		$payload['Headers'] = $headers;
 	}
 
+    /**
+     * Applies the message stream into the API payload.
+     *
+     * @param  array                     $payload
+     * @param  Swift_Mime_SimpleMessage  $message
+     * @return void
+     */
+    private function processMessageStream(&$payload, $message)
```

Comment:
Please don't make anything `private`, you don't know in what ways other users/customers need to customize things.
```suggestion
    protected function processMessageStream(&$payload, $message)
```

See also https://github.com/wildbit/swiftmailer-postmark/pull/33

Reply:
The methods in this code are all mostly private, i keep to the existing style. No need to combine #33 into this here.